helm/blazegraph/values.yaml

```diff
@@ -0,0 +1,61 @@
+# Default values for blazegraph.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: 141380700111.dkr.ecr.us-west-2.amazonaws.com/blazegraph
+  tag: newrelic
+  pullPolicy: Always
+
+persistence:
+  enabled: true
+  accessMode: ReadWriteOnce
+  size: 8Gi
+
+service:
+  type: ClusterIP
+  port: 8080
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  path: /
+  hosts:
+    - chart-example.local
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+#apps_sparql_async_server <- Analytic??
+analytic:
+  enabled: true
+  sparql_async_server: false
+
+java_ops:
+  memory: 1024
```

Comment:
do we need the java_ops?  Should this possibly pull from the resource limit memory as a fraction?  Certainly not immediately required, but it's something we are going to want with a java app like blazegraph

Reply:
Something that pulls resource limits makes sense long term. I think in the short term its not something to worry about. Currently this setting is also only used when the analytics server is enabled. It was placed here to test the logic of adding the config file. I can remove if requested.